invenio_notifications/backends/email/__init__.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
```

Comment:
minor: since we're not yet sure how big/complex backends can get, I would start small and keep everything in one file, i.e. `invenio_notifications/backends/email.py`.